docs/build/guides/basics/send-and-receive-payments.mdx

```diff
@@ -7,90 +7,251 @@ description: Learn to send payments and watch for received payments on the Stell
+// The source account is the one that will send the payment, and the destination account
+// is the one that will receive the payment.
+const sourceKeys = StellarSdk.Keypair.fromSecret(
+  "SCZANGBA5YHTNYVVV4C3U252E2B6P6F5T3U6MM63WBSBZATAQI3EBTQ4"
 );
-var destinationId = "GA2C5RFPE6GCKMY3US5PAB6UZLKIGSPIUKSLRB6Q723BM2OARMDUYEJ5";
-// Transaction will hold a built transaction we can resubmit if the result is unknown.
-var transaction;
+const destinationId =
+  "GA2C5RFPE6GCKMY3US5PAB6UZLKIGSPIUKSLRB6Q723BM2OARMDUYEJ5";
 
 // First, check to make sure that the destination account exists.
-// You could skip this, but if the account does not exist, you will be charged
-// the transaction fee when the transaction fails.
-server
-  .loadAccount(destinationId)
-  // If the account is not found, surface a nicer error message for logging.
-  .catch(function (error) {
-    if (error instanceof StellarSdk.NotFoundError) {
-      throw new Error("The destination account does not exist!");
-    } else return error;
-  })
-  // If there was no error, load up-to-date information on your account.
-  .then(function () {
-    return server.loadAccount(sourceKeys.publicKey());
-  })
-  .then(function (sourceAccount) {
-    // Start building the transaction.
-    transaction = new StellarSdk.TransactionBuilder(sourceAccount, {
-      fee: StellarSdk.BASE_FEE,
-      networkPassphrase: StellarSdk.Networks.TESTNET,
-    })
-      .addOperation(
-        StellarSdk.Operation.payment({
-          destination: destinationId,
-          // Because Stellar allows transaction in many currencies, you must
-          // specify the asset type. The special "native" asset represents Lumens.
-          asset: StellarSdk.Asset.native(),
-          amount: "10",
-        }),
-      )
-      // A memo allows you to add your own metadata to a transaction. It's
-      // optional and does not affect how Stellar treats the transaction.
-      .addMemo(StellarSdk.Memo.text("Test Transaction"))
-      // Wait a maximum of three minutes for the transaction
-      .setTimeout(180)
-      .build();
-    // Sign the transaction to prove you are actually the person sending it.
-    transaction.sign(sourceKeys);
-    // And finally, send it off to Stellar!
-    return server.submitTransaction(transaction);
-  })
-  .then(function (result) {
-    console.log("Success! Results:", result);
+try {
+  await rpc.getAccount(destinationId);
+} catch (error) {
+  console.error("Error checking destination account:", error);
+  throw error;
+}
+
+// Now we also load the source account to build the transaction.
+let sourceAccount: StellarSdk.Account;
+try {
+  sourceAccount = await rpc.getAccount(sourceKeys.publicKey());
+} catch (error) {
+  console.error("Error checking source account:", error);
+  throw error;
+}
+
+// The next step is to parametrize and build the transaction object
+let transaction: StellarSdk.Transaction;
+try {
```

Comment:
I think we can remove this try/catch, because all the errors would be unexpected rather than expected. For example, a 404 not found from Horizon when looking up the account is an expected error, hence the try/catch.